.circleci/config.yml

```diff
@@ -1,21 +1,19 @@
 version: 2.1
 
 orbs:
-  aws-ecr: circleci/aws-ecr@3.0.0
-  aws-cli: circleci/aws-cli@0.1.9
   aws_assume_role: lbh-hackit/aws_assume_role@0.1.0
-  sonarcloud: sonarsource/sonarcloud@1.1.1
+  sonarcloud: sonarsource/sonarcloud@2.0.0
 
 executors:
   docker-python:
     docker:
-      - image: cimg/python:3.8.4
+      - image: cimg/python:3.7
```

Comment:
Not sure why, but sonarcloud in docker doesn't work with python 3.8.4.
Downgraded to match other APIs.